docs/dev/documentation.md

```diff
@@ -4,38 +4,36 @@
 
 ## Building the docs
 
-Dependencies for building the documentation for scanpy can be installed with `pip install -e "scanpy[doc]"`
-
-To build the docs, enter the `docs` directory and run `make html`. After this process completes you can take a look at the docs by opening `scanpy/docs/_build/html/index.html`.
+To build the docs, run `hatch run docs:build`.
+Afterwards, you can run `hatch run docs:open` to open {file}`docs/_build/html/index.html`.
 
 Your browser and Sphinx cache docs which have been built previously.
 Sometimes these caches are not invalidated when you've updated the docs.
 If docs are not updating the way you expect, first try "force reloading" your browser page – e.g. reload the page without using the cache.
-Next, if problems persist, clear the sphinx cache and try building them again (`make clean` from `docs` directory).
-
-```{note}
-If you've cloned the repository pre 1.8.0, you may need to be more thorough in cleaning.
-If you run into warnings try removing all untracked files in the docs directory.
-```
+Next, if problems persist, clear the sphinx cache (`hatch run docs:clean`) and try building them again.
 
 ## Adding to the docs
 
-For any user-visible changes, please make sure a note has been added to the release notes for the relevant version so we can credit you!
-These files are found in the `docs/release-notes/` directory.
+For any user-visible changes, please make sure a note has been added to the release notes using `hatch run towncrier:create`.
```

Comment:
Probably a good idea to link to this https://towncrier.readthedocs.io/en/stable/tutorial.html#creating-news-fragments since there is no documentation here about how to use this

Reply:
I was thinking that people can use `--help`, but sure!

Reply:
Ah, good point.  Had not considered that.  Still think it's nice to add here.